mz_zip_rw.h

```diff
@@ -102,7 +102,7 @@ int32_t mz_zip_reader_save_all(void *handle, const char *destination_dir);
 
 /***************************************************************************/
 
-void mz_zip_reader_set_pattern(void *handle, const char *pattern, uint8_t ignore_case);
+int32_t mz_zip_reader_set_pattern(void *handle, const char *pattern, uint8_t ignore_case);
```

Comment:
If you prefer that I avoid changing the signature here, then I can add a new api named `mz_zip_reader_copy_pattern` or whatever name you want.

Reply:
I don't mind it changed. Can you check to see if it needs to be updated in the `docs` folder?

Reply:
https://github.com/zlib-ng/minizip-ng/pull/874/commits/84a1b033d2c1bd84edaf0f68ab9f5a187aa60d87